staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -989,7 +989,6 @@ var (
 	}
 	etpLocalNodePredicates []NodeConditionPredicate = []NodeConditionPredicate{
 		nodeIncludedPredicate,
-		nodeUnTaintedPredicate,
```

Comment:
based on my read of cluster-autoscaler, I think we need to keep this.  Let's let that discussion play out.